meetings/2023-11-29.md

```diff
@@ -0,0 +1,90 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2023-11-29
+
+## Links
+
+* **Recording**:
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1476>
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1rIyGl0PyVORxRF_0hEDGZd-qdNKU5OnmJbAgME7zbww/edit>
+
+## Present
+
+* Michaël Zasso @targos (voting member)
+* Yagiz Nizipli @anonrig (voting member)
+* Gireesh Punathil @gireeshpunathil (voting member)
+* Joyee Cheung @joyeecheung (voting member)
+* Chengzhong Wu @legendecas (voting member)
+* Matteo Collina @mcollina (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Darshan Sen @RaisinTen (voting member)
+* Robert Nagy @ronag (voting member)
+* Richard Lau @richardlau (voting member)
+* Antoine du Hamel @aduh95 (voting member)
+* Ruben Bridgewater @BridgeAR (voting member)
+
+## Agenda
+
+### Announcements
+
+\*Node.js v21.3.0 postponed because of bugs
+\*Node.js v18.x last “big release” with 364 commits
+-it will ship the new off-thread ESM module loader (if using ts-node you will need to change)
```

Comment:
```suggestion
* Node.js v21.3.0 postponed because of bugs
* Node.js v18.x last “big release” with 364 commits
* it will ship the new off-thread ESM module loader (if using ts-node you will need to change)
```